Demo/Demo/Models/AccessTokenRequest.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
```

Comment:
Should these files only be in the Demo app now? Are they not required in the main framework? Sorry if this is a silly question - couldn't figure it out from the PR description

Reply:
Yes, these are communicating with sample merchant server not paypal api
so merchants will have their own thing to communicate with their servers

Reply:
Would it be possible for you to explain that a bit more in the PR description? Why these changes are made/what merchants are fetching instead of us?

Reply:
Works for me, thanks!